terraform/modules/api_gateway/gateway/variables.tf

```diff
@@ -52,13 +52,13 @@ variable "api_gateway_usage_plans" {
   default = {
     internal_apps = {
       quota_limit = 10000 # per day
-      burst_limit = 100   # per second
-      rate_limit  = 200
+      burst_limit = 300
```

Comment:
Maybe increase quota_limit and burst_limit even more (50000/1000)?  Kinshuk said we need high limits for processing of large lists - there could be a lot of requests if a customer creates a new large lists with 25000 locations.

Reply:
@danscales: adding this to develop branch, merged before adding.